lib/disk/modules/AzureCommon.rb

```diff
@@ -0,0 +1,95 @@
+require_relative '../MiqDisk'
+require 'ostruct'
+
+module AzureCommon
```

Comment:
@jerryk55 I think this name isn't specific enough. Would `AzureDiskCommon` be better?